spring/src/main/java/io/cloudevents/spring/http/CloudEventsHeaders.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018-Present The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.cloudevents.spring.http;
+
+import java.util.Map;
+
+import io.cloudevents.core.message.impl.MessageUtils;
+
+import org.springframework.http.HttpHeaders;
+
+public class CloudEventsHeaders {
+
+	public static final String CE_PREFIX = "ce-";
+
+	public static final Map<String, String> ATTRIBUTES_TO_HEADERS = MessageUtils
```

Comment:
minor thing: I could see the maintenance of this mapping as part of the `MessageUtils`, too.

Reply:
Is there some action required in the PR? Not sure what you mean really.